bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx71.html

```diff
@@ -0,0 +1,120 @@
+
+{% set _entrypoint = 'mozilla.org-whatsnew71' %}
+{% set _utm_campaign = 'whatsnew71' %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+<main class="content-wrapper mzp-t-firefox mzp-t-dark">
+  <header class="c-page-header">
+    <div class="mzp-l-content c-page-header-inner">
+      {{ high_res_img('protocol/img/logos/firefox/browser/logo-word-hor-white-sm.png', {'alt': 'Firefox', 'width': '216', 'height': '40', 'class': 'c-page-header-logo-fx'}) }}
+      <div class="mzp-c-notification-bar mzp-t-success show-firefox-desktop-70">
+        <p>{{ _('Congrats! You’re using the latest version of Firefox.') }}</p>
+      </div>
+    </div>
+  </header>
+
+  <section class="content-main">
+    <div class="mzp-l-content">
+      <div class="mzp-c-emphasis-box">
+        <img class="c-emphasis-box-logo" src="{{ static('protocol/img/logos/firefox/logo-md.png') }}" alt="">
+        <h2 class="c-emphasis-box-title">
+          {{ _('No account required. But you might want one.') }}
+        </h2>
+        <p>{{ _('The Firefox browser collects so little data about you, we don’t even require your email address. But when you use it to create a Firefox account, we can protect your privacy across more of your online life.') }}</p>
+
+        <p class="cta">{{ fxa_cta_link(entrypoint=_entrypoint, action='signup', button_text=_('Sign In'), button_class='mzp-c-button mzp-t-product mzp-t-small js-fxa-product-button', utm_campaign=_utm_campaign, utm_content='top-cta') }}</p>
+      </div>
+    </div>
+  </section>
+
+  <section class="content-extra">
+    <div class="mzp-l-content">
+
+      <div class="l-columns-one c-picto-list">
+        <div class="c-picto-block t-adjacent">
+          <div class="c-picto-block-image">
+            <img src="{{ static('img/logos/firefox/logo-monitor.svg') }}" alt="">
+          </div>
+          <h3 class="c-picto-block-title">Firefox Monitor</h3>
+          <div class="c-picto-block-body">
+            <p>{{ _('Have at least one company looking out for your data, instead of leaking it.') }}</p>
+          </div>
+        </div>
+
+        <div class="c-picto-block t-adjacent">
+          <div class="c-picto-block-image">
+            <img src="{{ static('img/logos/firefox/logo-lockwise.svg') }}" alt="">
+          </div>
+          <h3 class="c-picto-block-title">Firefox Lockwise</h3>
+          <div class="c-picto-block-body">
+            <p>{{ _('Never forget, reset or travel without your passwords again.') }}</p>
+          </div>
+        </div>
+
+        <div class="c-picto-block t-adjacent">
+          <div class="c-picto-block-image">
+            <img src="{{ static('img/logos/fbcontainer/logo-fbcontainer.svg') }}" alt="">
+          </div>
+          <h3 class="c-picto-block-title">Facebook Container</h3>
+          <div class="c-picto-block-body">
+            <p>{{ _('Get an airtight container to keep Facebook out of your business.') }}</p>
```

Comment:
We removed "airtight" from this copy for welcome 3 and should do the same here. New version is:

`Get a container to keep Facebook out of your business.`